python/tests/integration/arcticdb/test_lmdb.py

```diff
@@ -102,6 +102,30 @@ def test_lmdb_malloc_trim(lmdb_storage):
     lib = ac.create_library("test_lmdb_malloc_trim")
     lib._nvs.trim()
 
+@pytest.mark.skipif(sys.platform != "win32", reason="Non Windows platforms have different file path name restrictions")
+@pytest.mark.parametrize("invalid_lib_name", ["lib?1", "lib:1", "lib|1", "lib.", "lib "])
```

Comment:
This testing is not comprehensive of all of the disallowed characters in `invalid_win32_chars`

Reply:
The chars `*<>` are already disallowed for libraries in all storages [here](https://github.com/man-group/ArcticDB/blob/master/cpp/arcticdb/util/name_validation.cpp#L123) and tested [here](https://github.com/man-group/ArcticDB/blob/master/python/tests/integration/arcticdb/test_arctic_library_management.py#L243).